src/SignalR/common/SignalR.Common/test/Internal/Protocol/JsonHubProtocolTests.cs

```diff
@@ -101,6 +101,23 @@ public void MagicCast()
             Assert.Equal(expectedMessage, message);
         }
 
+        [Fact]
+        public void ReadCaseInsensitivePropertiesByDefault()
+        {
+            var input = Frame("{\"type\":2,\"invocationId\":\"123\",\"item\":{\"StrIngProp\":\"SignalR!\",\"DoublePrOp\":6.2831853071,\"IntProp\":43,\"DateTimeProp\":\"2017-04-11T00:00:00Z\",\"NuLLProp\":null,\"ByteARRProp\":[1,2,3]}}");
+
+            var binder = new TestBinder(null, typeof(TemporaryCustomObject));
+            var data = new ReadOnlySequence<byte>(Encoding.UTF8.GetBytes(input));
+            JsonHubProtocol.TryParseMessage(ref data, binder, out var message);
+
+            var streamItemMessage = Assert.IsType<StreamItemMessage>(message);
+            Assert.Equal(new TemporaryCustomObject()
+            {
+                ByteArrProp = new byte[] { 1, 2, 3 },
```

Comment:
Why don't you check any of the other properties here?

Reply:
I guess there is no harm in changing all properties, I'll update :+1: